src/main/java/org/broadinstitute/consent/http/configurations/ServicesConfiguration.java

```diff
@@ -19,6 +19,8 @@ public class ServicesConfiguration {
 
   private Integer poolSize = 10;
 
+  private Integer cacheExpireMinutes = 1;
```

Comment:
Is it weird to have this in minutes but you've got `timeoutSeconds` right above?

Reply:
I can certainly change it for consistency, but they are conceptually very different things.

Reply:
Maybe some additional commentary around each value would be helpful

Reply:
Yeah, that's totally fair.  And there's likely no need to have a more exact cache duration than minutes.

Re docs, that's maybe more general than this PR.  But yeah, maybe java docs around the service config might be useful. Totally not a blocker to merge BTW